tool/src/query-to-elm.ts

```diff
@@ -90,8 +99,14 @@ let moduleName = 'GraphQL.' + filename;
 
 let outPath = path.join(path.dirname(graphqlFile), filename + '.elm');
 
-let url = introspectionUrl + '?query=' + encodeURIComponent(introspectionQuery.replace(/\n/g, '')); 
-request(url, function (err, res, body) {
+let url = introspectionUrl;
+let data = {query: introspectionQuery.replace(/\n/g, '')};
+let method = options["method"] || "GET";
+let reqopts = method == "GET" ? {url:url, method:method, qs:data}
+                              : {url:url, method:method,
+                                 headers: [{"Conent-Type": "application/json"}],
```

Comment:
Oh, I'm embarrassed! For some reason Reindex didn't care about the malformed header and the typo got unnoticed 😞